WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -109,6 +108,13 @@ private extension ItemListView {
         .padding(.horizontal, Constants.bannerCardPadding)
     }
 
+    private var bannerHintAndLearnMoreText: Text {
+        Text(Localization.headerBannerHint + " ") +
+        Text(Localization.headerBannerLearnMoreHint)
+            .font(.body)
```

Comment:
Both the text and learn more button should be `posDetailEmphasized`

```suggestion
            .font(.posDetailEmphasized)
```

Reply:
Fixed! https://github.com/woocommerce/woocommerce-ios/pull/13940/commits/7acf98a1552e71d17f5e7233d3f17d95e28690d7 , interestingly we have to declare the type and cast to `font()` or the types don't match and the compiler is unhappy.

Reply:
Yes, interesting. I played around and something like this worked. Likely related to the fact that we're appending two Texts here.

```
    private var bannerHintAndLearnMoreText: some View {
        (Text(Localization.headerBannerHint + " ") +
        Text(Localization.headerBannerLearnMoreHint))
            .font(.posDetailEmphasized)
            .foregroundColor(Color(.accent))
    }
```